homeassistant/components/mysensors/const.py

```diff
@@ -107,14 +143,25 @@
     "sensor": SENSOR_TYPES,
     "switch": SWITCH_TYPES,
 }
+"""dict mapping hass platform name to dict that maps mysensors s_type to set of mysensors v_type"""
```

Comment:
Comments are done with `# `.

Reply:
Those are supposed to be docstrings for these constants.
I know [PEP-257](https://www.python.org/dev/peps/pep-0257/#id15) says they are not "real docstrings" in the sense that they will be available as `__doc__`, but they are are allowed and considered "additional docstrings". And there are tools that read them: For example the PyCham IDE displays them whenever you hover over any usage of the constant.

Are you sure you want me to convert them to normal comments?

Reply:
Yes, please. Personally I don't think we need this many comments.